models/notification.go

```diff
@@ -124,6 +126,25 @@ func (message *Notification) checkConfig() {
 	message.Recipients = newRecipient
 }
 
+var (
+    reHole    = regexp.MustCompile(`#{1,2}\d+`)
+    reFormula = regexp.MustCompile(`\${1,2}.*?\${1,2}`)
```

Comment:
It does not match a multi-line string.

Play it here: <https://go.dev/play/p/O3g5Lz7W6Ta>